packages/pnpm-sync-lib/etc/pnpm-sync-lib.api.md

```diff
@@ -58,12 +64,16 @@ export interface IPnpmSyncCopyOptions {
 }
 
 // @beta (undocumented)
-export interface IPnpmSyncPrepareOptions {
+export interface IPnpmSyncPrepareBaseOptions {

```

Comment:
`IPnpmSyncPrepareBaseOptions` -> `IPnpmSyncUpdateFileBaseOptions`

because "updateFile" is the more primitive operation, whereas "prepare" actually calls "updateFile"